pkg/consts/consts.go

```diff
@@ -230,6 +230,9 @@ const (
 	// ServiceAnnotationPIPName specifies the pip that will be applied to load balancer
 	ServiceAnnotationPIPName = "service.beta.kubernetes.io/azure-pip-name"
 
+	// ServiceAnnotationPIPPrefixID specifies the pip prefix that will be applied to the load balancer.
+	ServiceAnnotationPIPPrefixID = "service.beta.kubernetes.io/azure-pip-prefix-id"
```

Comment:
minor concern: will it be good to use name instead of id to keep aligned with the ServiceAnnotationPIPName?

Reply:
I think id is more general. Adding prefix-name would also need another prefix-resource-group, and maybe also its subscription, hence three annotations would be needed instead of one.